_posts/2017-11-23-Rust-1.22.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "Announcing Rust 1.22"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.22.0. Rust
+is a systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.22 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.22.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1220-2017-11-23
+
+## What's in 1.22.0 stable
+
+The headline feature for this release is one many have been waiting for
```

Comment:
"waiting for for" sounds a bit awkward, what about "is one many have been anticipating for a long time"? "is one that has been widely anticipated for a long time"?